jekyll/_cci2/configuration-reference.md

```diff
@@ -650,7 +650,7 @@ Note the following distinctions between Artifacts, Workspaces, and Caches:
 | Type      | lifetime        | Use                      | Example |
 |-----------|-----------------|------------------------------------|---------
 | Artifacts | Months          | Preserve long-term artifacts. |  Available in the Artifacts tab of the Build details under the `tmp/circle-artifacts.<hash>/container`   or similar directory.     |
-| Workspaces | Duration of workflow        | Move temporary files around for use by jobs run later in the workflow. | The `persist_to_workspace: <directory>` key under jobs, refer to the file with `attach_workspace: at: <directory>`.    |
+| Workspaces | Duration of workflow        | Move temporary files around for use by jobs run later in the workflow. | The `persist_to_workspace:` key under jobs, refer to the file with the `attach_workspace:` key.    |
```

Comment:
It would be more accurate to say "attach the workspace in a downstream container with the `attach_workspace:` step."

`attach_workspace` copies and re-creates the entire workspace content when it's run.